syntax = "proto3";
package visualInputNode;

import "google/api/annotations.proto";

/* Visual Input Node Service */

service VisualInputNodeService {

// <------------------- Initialization -------------------------------->
  rpc InitializeVisualInputNode (InitializeVisualInputNodeRequest) returns (InitializeVisualInputNodeResponse);
}


message InitializeVisualInputNodeRequest {

}


// Returns the playlists and directories
message InitializeVisualInputNodeResponse {
  bool isInitialized = 1;

}
